sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/CheckpointFileManager.scala

```diff
@@ -83,6 +83,9 @@ trait CheckpointFileManager {
 
   /** Is the default file system this implementation is operating on the local file system. */
   def isLocal: Boolean
+
+  /** Returns the qualified path of the checkpoint path. */
+  def getQualifiedCheckpointPath(): Path
 }
 
```

Comment:
not related to this PR: I think it's better to document clearly that whether the path parameter should be relative or qualified, in all the methods of this trait, even if this is just an internal API.

Reply:
Good point. I roughly looked at few places, seems the used parameter is absolute path, though not qualified.